test/babelish/test_android2csv.rb

```diff
@@ -21,15 +21,39 @@ def test_initialize
       :csv_filename => csv_filename,
       :headers => headers,
       :filenames => filenames
-      })
+    })
 
-      assert_equal csv_filename, converter.csv_filename
-      assert_equal headers, converter.headers
-      assert_equal filenames, converter.filenames
-    end
+    assert_equal csv_filename, converter.csv_filename
+    assert_equal headers, converter.headers
+    assert_equal filenames, converter.filenames
+  end
 
-    def test_initialize_with_default_values
-      converter = Babelish::Android2CSV.new
-      assert_not_nil converter.csv_filename
-    end
+  def test_initialize_with_default_values
+    converter = Babelish::Android2CSV.new
+    assert_not_nil converter.csv_filename
+  end
+
+  def test_special_chars
+    csv_filename = "./android_special_chars_output.csv"
+    filenames = "test/data/android_special_chars.xml"
+    headers = %w{"constants german"}
+
+    converter = Babelish::Android2CSV.new({
+      :csv_filename => csv_filename,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.